coordinator/src/routes/health/get-metrics.ts

```diff
@@ -0,0 +1,26 @@
+import Hapi from "@hapi/hapi"
+import * as osu from "node-os-utils"
+
+export default [
+  {
+    method: "GET",
+    path: "/_metrics",
+    handler: async (request: Hapi.Request, h: Hapi.ResponseToolkit): Promise<Hapi.ResponseObject> => {
+      const cpu = osu.cpu
```

Comment:
Not that it matters (the compiler will probably take out the interim field for you) but is there any reason we store a reference to `cpu` then call info on that instead of just doing `osu.cpu.info()`?